.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Question
+about: Ask a question about the project
+title: '[QUESTION] '
+labels: ['question']
+assignees: ''
+---
+
+## Question
+
+**What would you like to know about Compose?**
+
+### Your Question
+<!-- Ask your question in a clear and concise way -->
+
+### Context
+<!-- Provide any relevant context or background information -->
+
+### What I've Tried
+<!-- Describe what you've already tried or researched -->
+
+### Expected Outcome
+<!-- What would help you the most? -->
+
+### Code Example
+<!-- If applicable, provide a code example that relates to your question -->
+
+## Helpful Information
+
+**Have a question?** Please check our [README](https://github.com/Perfect-Abstractions/Compose/blob/main/README.md) and [CONTRIBUTING](https://github.com/Perfect-Abstractions/Compose/blob/main/CONTRIBUTING.md) files first - your answer might already be there!
```

Comment:
Already mentioned,  but especially for this issue,   the comment to check the CONTRIBUTORS.md file and also in this case the READM before asking a questions,   and also can check the discussions.